packages/cli/src/parse-args.ts

```diff
@@ -477,6 +479,13 @@ export const commands: Command[] = [
         ...message,
         description:
           "Message to comment on PR with. Defaults to 'Published PR with canary version: %v'. Pass false to disable the comment"
+      },
+      {
```

Comment:
We will also need support for `shipit`.  We could do this a few different ways:

1. change flag name to `--force-canary`. then both `canary` and `shipit` can use the same flag. Users would then be able to supply the value in .autorc `forceCanaries: true`
2. just add that flag to `shipit` also
3. have add the ability to target specific commands to autorc:

```jsonc
{
  // Configure args for specific commands
  "canary": {
     "force": true
  }
}
```

Thoughts?

Reply:
Yep, all that sounds great. Will update the PR tomorrow.

Reply:
With #998 merged we should be able to enable this pretty easily. Option `3` is now possible you just need to add `config: true` to `force` flag. This means both `1` and `2` are unneeded 🎉